CODE_OF_CONDUCT.md

```diff
@@ -1,66 +1,43 @@
-* Grunt - 
-* HospitalRun - `hello@hospitalrun.io`
-* Interledger.js - 
-* Intern - 
-* JerryScript - 
-* jQuery - 
-* libuv - 
-* LoDash - 
-* messageformat - 
-* Marko - 
-* Mocha - `report@lists.openjsf.org`
-* Moment.js - 
-* Node.js - `report@nodejs.org`
-* Node-RED - `team@nodered.org`
-* nvm - 
-* Pep - 
-* QUnit - 
-* Webdriver.io - [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
-* Webhint - `support@webhint.io`
-* webpack - [redirects here]
-
-### Foundation Spaces
-
-For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization, use the email `report@lists.openjsf.org`. The Cross Project Council (CPC) is responsible for managing these reports and commits to:
-
-- maintain the confidentiality with regard to the reporter of an incident
-- to participate in the path for escalation as outlined in
-  the section on Escalation when required.
-
-## Escalation
-
-The OpenJS Foundation maintains a Code of Conduct Panel (CoCP). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled. In order to escalate to the CoCP send an email to `coc-escalation@lists.openjsf.org`.
+The OpenJS Foundation and its member projects use [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct) in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
 
+## Commitment
+
+All recipients of reports commit to:
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
+
+
+## Report an issue in a project
+
+1. To report an issue in one of the projects listed below, please use the method provided. The project itself is responsible for managing these reports.
+    * **AMP Project:** <code-of-conduct@amp.dev>
+    * **Appium:** email maintainers
+    * **Dojo:** [Fill this reporting form](https://docs.google.com/forms/d/1ZiCUv3FsarWetoQ8J5QnCdIrHLj0xMmwNSLCyBkio7c/viewform?edit_requested=true)
+    * **Electron:** <coc@electronjs.org>
+    * **Express.js:** <express-coc@lists.openjsf.org>
+    * **Fastify:** <hello@matteocollina.com> or <tommydelved@gmail.com>
+    * **HospitalRun:** <hello@hospitalrun.io>
+    * **Node.js:** <report@nodejs.org>
+    * **Node-RED:** <team@nodered.org>
+    * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
+    * **Webhint:** <support@webhint.io>
+2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
```

Comment:
This is an interesting addition which I'm not sure we've discussed. That the CPC is responsible for taking reports for projects that don't have a mechanism that makes sense to me but we might want to add as a responsibility in https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-5-responsibilities-and-expectations-of-the-cpc.  We currently have this 
"Supporting projects in their enforcement of Code of Conduct violations and escalation." but what is captured by this line I think goes beyond that.

Reply:
@openjs-foundation/cpc FYI for broader visibliity.

Reply:
See proposed CPC charter change in 30239fef21a7a01d0c9595a6b1787def200ebb48.